.github/workflows/tests.yml

```diff
@@ -17,6 +17,9 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
```

Comment:
Not sure it is doable, but the building of release artifacts also maybe could be broken out? That doesn't take too long though.